DataPerProcessing/utilities.py

```diff
@@ -8,7 +8,33 @@
 imagePath = "..//..//DOTA20DataSets//train//images"
 labelPath = "..//..//DOTA20DataSets//train//labels"
 
-imageName = "P0000.png"
```

Comment:
you are initialising one variable two times???, why 
In drawboundingboxes and polygonToRotRectangle function is not in use, please remove it, taking unnecessary space.